### PR TITLE
Minimum element in a rotated array

### PR DESCRIPTION
**Pull request by ErzaTitania-2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

(Write your answer here.)

### Checklist

- [y ] I've read the contribution guidelines.
- [y ] I've checked the issue list before deciding what to submit.
- [y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Rotated Minimum #1835

### CODE REVIEW

**Comment by ErzaTitania-2001** [comment]
@iamrajiv

File: Java/cp/Min_Element.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import java.util.*;
```
Comment (iamrajiv) [inline Java/cp/Min_Element.java]: @ErzaTitania-2001 Add problem link.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ErzaTitania-2001 Rename the program to `min_element_rotated_array.java`.

File: Java/cp/Min_Element.java (hunk 1 above)
Comment (ErzaTitania-2001) [reply Java/cp/Min_Element.java]: Done. @iamrajiv

**Review by iamrajiv — APPROVED** [review]

**Comment by ErzaTitania-2001** [comment]
@HarshCasper review?

**Comment by ErzaTitania-2001** [comment]
@iamrajiv can this be merged so that i can move onto another PR?

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**